### ISSUE TITLE
Tags containing only '<' cause Parsing Error 

### ISSUE THREAD

**Issue by IdrisN** [vuejs/eslint-plugin-vue#1091]
**Tell us about your environment**	
* **ESLint version: 6.8.0**	
* **eslint-plugin-vue version: 6.1.2**	
* **Node version: 12.12.0**	

**What did you do?**	

```html
<span><</span>
```

**What did you expect to happen?**
This should be parsed correctly.

**What actually happened?**
```Parsing error: invalid-first-character-of-tag-name  vue/no-parsing-error```

Remark: ```<span>></span> ``` ist parsed correctly.

**Comment by IdrisN** [comment]
I just realised, that although it works, it's illegal html markup. 
https://html.spec.whatwg.org/multipage/parsing.html#parse-error-invalid-first-character-of-tag-name

One should use `&lt;`.
